lib/STREAMS.md

```diff
@@ -132,3 +132,19 @@ Can return synchronously. Use `await` or wrap the return value in `Promise.resol
 ### readStream.peekBuffer(byteCount)
 
 Like `readStream.read` and `readStream.peek`, but returns a Buffer instead of a string.
+
+### readStream.peeker([encoding])
+### readStream.reader([encoding])
```

Comment:
I've been considering renaming the regular read methods to `readChunk` etc, so it's clearer that if you're not doing a delimited read, the boundaries are meaningless.

Reply:
On the other hand, for `ObjectReadStream`s, you can probably just do `for await (const chunk of stream)` directly, skipping the `.reader()`.

Reply:
I think `readChunk` would be clearer, yeah. I'm not sure how I feel about `for await (const chunk of stream)` for `ObjectReadStream` only though...it would make the API less consistent, but at the same time `reader` doesn't share a type between the two anyway.

Reply:
I would also call it `chunkReader` to be in line with `readChunk`. I think reading an arbitrary (or fixed-size) chunk from a ReadStream is just an entirely different operation from reading exactly one element from an ObjectReadStream.